test/unit/api-test.js

```diff
@@ -58,43 +58,46 @@ describe('Api', function() {
         assert.equal(check, false, 'api.off works');
     });
 
-    it('bad events don\'t break player', function() {
-        jwplayer.debug = false;
+    it('bad events doesn\'t break player', function() {
+        var tmpConsole = console.log;
+
+        console.log = function() {};
 
         const api = createApi('player');
-        let check = false;
 
-        function update() {
-            check = true;
+        function validEvent() {
+            throw new TypeError('blah');
         }
 
-        function bad() {
+        function invalidEvent() {
             throw new TypeError('blah');
         }
 
-        api.on('x', bad);
-        api.on('x', update);
-        api.on('x', bad);
+        api.on('x', invalidEvent);
+        api.on('x', validEvent);
+        api.on('x', invalidEvent);
 
-        api.trigger('x');
+        expect(function() {
+            api.trigger('x');
+        }).to.not.throw();
 
-        assert.isOk(check, 'When events blow up, handler continues');
```

Comment:
You've changed the functionality of this test by removing the update method. Why did you make `validEvent()` and `invalidEvent()` the same?